tests/jsx-significant-space/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -90,26 +90,26 @@ before = (
 
 before_break1 = (
   <span>
-    <span barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar />
-    {" "}foo
+    <span
+      barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar
+    />{" "}
+    foo
   </span>
 );
 
 before_break2 = (
   <span>
     <span
       barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar
-    />
-    {" "}foo
+    />{" "}
```

Comment:
`before_break{1,2}` can go either way for me (I do prefer this new version, though), but `after_break`looks way better, and I also think `regression_not_transformed_2` is more readable. So I'm really digging this change! 😁